x-pack/auditbeat/module/system/socket/_meta/docs.asciidoc

```diff
@@ -4,3 +4,128 @@ beta[]
 
 This is the `socket` dataset of the system module.
```

Comment:
Seems like this comment got dropped by Github. Rewriting it...

Can you add an overview of the features this provides (like the selling points). It might be a little redundant with the second paragraph below, but that's ok. Like UDP and TCP flow tracking for ipv4/ipv6, records the user and process associated with the flow, and does not require a kernel module

Reply:
Done. Please have a look